Log Viewing/ARKLogTableViewController.m

```diff
@@ -54,7 +55,10 @@ @implementation ARKLogTableViewController
 
 - (instancetype)initWithLogStore:(ARKLogStore *)logStore logFormatter:(id <ARKLogFormatter>)logFormatter;
 {
-    self = [super init];
+    NSAssert(logStore, @"Must pass a log store.");
+    NSAssert(logFormatter, @"Must pass a logFormatter.");
```

Comment:
ARKCheckCondition

A logging framework should not cause a crash. Do not use asserts